streams/service/src/main/java/com/hortonworks/streamline/streams/service/TopologyTestRunResource.java

```diff
@@ -736,4 +774,115 @@ public Long getTimestamp() {
         }
     }
 
+    // FIXME: this class is for this service, not runtime
+    public static class EventInformation {
```

Comment:
We may want to unify this and runtime-storm's EventInformation. There're some other classes which are not strictly tied to runtime-storm but placed to runtime-storm. I'll see how we can address it.